File: king-admin-java/src/main/java/com/oukingtim/service/SysUserService.java
package com.oukingtim.service;

import com.baomidou.mybatisplus.service.IService;
import com.oukingtim.domain.SysUser;

/**
 * Created by oukingtim
 */
public interface SysUserService extends IService<SysUser> {

    /**
     * 根据用户名查询用户
     * @param username
     * @return
     */
    SysUser findByUserName(String username);
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/service/SysUserService.kt
package com.oukingtim.service

import com.baomidou.mybatisplus.service.IService
import com.oukingtim.domain.SysUser

/**
 * Created by oukingtim
 */
interface SysUserService : IService<SysUser> {
    /**
     * 根据用户名查询用户
     * @param username
     * *
     * @return
     */
    fun findByUserName(username: String): SysUser
}